internal/capability/auditor.go

```diff
@@ -46,9 +47,11 @@ type CapAuditor struct {
 // │   ├── manifest_file1.json
 // │   └── manifest_file2.yaml
 // └── package_name2
-//     ├── manifest_file1.json
+//
+//	   ├── manifest_file1.json
+//
```

Comment:
I think that change was unintended ?

Haven't reviewed the rest.

Reply:
You are correct. I gotta figure out a way to make gofumpt ignore that.